src/BerlinVegan.cpp

```diff
@@ -75,15 +76,18 @@ int main(int argc, char *argv[])
 #endif
 
     qmlEngine.addImportPath(QStringLiteral("qrc:/imports/"));
+    FileIO fileIO;
 
 #ifdef Q_OS_SAILFISH
     app->setApplicationVersion(APP_VERSION);
     view->setSource(mainQMLFile);
     view->show();
+    view->rootContext()->setContextProperty("FileIO", &fileIO);
```

Comment:
Why did you take the context property approach --- as opposed to making
```FileIO``` a static class? This class doesn't have any state, does it?

Reply:
It is just the way, they do it in the example I mentioned in one of my commits: https://askubuntu.com/a/454998

And I have Babo's voice in my head sating to avoid static classes (although we are not going to run our app multiple times on the same device). Can we leave it the way it is or are there any disadvantages?

Reply:
Sure we can leave it like that. BTW: Static classes wouldn't hurt here, as this class has no state. You could even run it multiple times.